src/basicOp.js

```diff
@@ -24,7 +24,18 @@
  * @return {number}
  */
 function basicOp(operation, value1, value2) {
-  // write code here
+  switch (operation) {
+    case '+' :
```

Comment:
Пробел между кейсом и двоеточием не нужен
```suggestion
    case '+':
```